css/types/flex.json

```
@@ -5,6 +5,7 @@
         "__compat": {
           "description": "<code>&lt;flex&gt;</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flex_value",
+          "spec_url": "https://drafts.csswg.org/css-grid/#typedef-flex",
```

```suggestion
          "spec_url": "https://drafts.csswg.org/css-grid/#fr-unit",
```

That's the section containing the definition. But it shows that we're missing the spec URL for the fr unit entry in BCD.